app/views/case_contacts/_form.html.erb

```diff
@@ -141,20 +148,22 @@
           end
         %>
         <% if message %>
-          <%= form.text_field :no_address_content, class: "form-control red-letter", value: message, disabled: true,
+        <!--          TODO what does this look like?????        -->
+          <%= form.text_field :no_address_content, class: "input-style-1 red-letter", value: message, disabled: true,
 readonly: true %>
         <% else %>
           <%= form.fields_for :casa_case, casa_cases[0] do |cc| %>
             <%= cc.fields_for :volunteers, volunteer do |v| %>
               <%= v.fields_for :address, address do |a| %>
-                <%= a.text_field :content, class: "form-control", required: true %>
+                <%= a.text_field :content, class: "input-style-1", required: true %>
               <% end %>
             <% end %>
           <% end %>
         <% end %>
+        </div>
       </div>
     <% end %>
-
+<!--          TODO what does this look like????          -->
```

Comment:
You can set it to true in the database or just remove the if check all together just to see in the browser.